.github/workflows/pr_approval.yml

```diff
@@ -5,18 +5,19 @@ name: Check PR Approvals
 # To protect against that scenario, we can turn on number of approvals required to 2 in the github settings
 # of the repository
 on:
+  workflow_dispatch:
```

Comment:
This doesn't run in the context of a PR though, does it? You could add an option to provide a PR link or a PR number. But I don't know if it's worth it